roles/ood_user_reg_cloud/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
   git:
     repo: "{{ user_register_app_repo }}"
     dest: "{{ user_register_app_path }}"
-    version: "{{ user_register_app_tag }}"
+    version: master
```

Comment:
I think this should still be a var, may be just rename it to `user_register_app_version` ??

Reply:
I concur, user_register_app_tag in  group_vars/all needs to be changed to master